Building without sudo

Hi, is there any way to get this to work without sudo? Travis seems to be making a big push to get people to not use sudo anymore.

It's the `sudo make install` part that doesn't seem to work without sudo:

```bash
-- Installing: /usr/local/bin/kcov

CMake Error at src/cmake_install.cmake:42 (FILE):

  file INSTALL cannot copy file

  "/home/travis/build/TheDan64/inkwell/kcov-master/build/src/kcov" to

  "/usr/local/bin/kcov".

Call Stack (most recent call first):

  cmake_install.cmake:37 (INCLUDE)

make: *** [install] Error 1
```

I guess it's trying to copy to the /usr/ directory for some reason

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,5 +1,7 @@
 language: rust
 
+sudo: false
+
 rust:
   - stable
   - beta
@@ -12,9 +14,6 @@ matrix:
 env:
   global:
   - RUSTFLAGS="-C link-dead-code"
-
-before_install:
-  - sudo apt-get update
 
 addons:
   apt:
@@ -34,10 +33,9 @@ after_success: |
   cd build &&
   cmake .. &&
   make &&
-  sudo make install &&
+  make install DESTDIR=../../kcov-build &&
   cd ../.. &&
   rm -rf kcov-master &&
-  for file in target/debug/examplerust-*[^\.d]; do mkdir -p "target/cov/$(basename $file)"; kcov --exclude-pattern=/.cargo,/usr/lib --verify "target/cov/$(basename $file)" "$file"; done &&
+  for file in target/debug/examplerust-*[^\.d]; do mkdir -p "target/cov/$(basename $file)"; ./kcov-build/usr/local/bin/kcov --exclude-pattern=/.cargo,/usr/lib --verify "target/cov/$(basename $file)" "$file"; done &&
   bash <(curl -s https://codecov.io/bash) &&
   echo "Uploaded code coverage"
-

--- a/README.md
+++ b/README.md
@@ -67,6 +67,8 @@ of Rust you desire.
 ```yml
 language: rust
 
+sudo: false
+
 rust:
   - stable
   - beta
@@ -79,9 +81,6 @@ matrix:
 env:
   global:
   - RUSTFLAGS="-C link-dead-code"
-
-before_install:
-  - sudo apt-get update
 
 addons:
   apt:
@@ -101,10 +100,10 @@ after_success: |
   cd build &&
   cmake .. &&
   make &&
-  sudo make install &&
+  make install DESTDIR=../../kcov-build &&
   cd ../.. &&
   rm -rf kcov-master &&
-  for file in target/debug/<PROJECT-NAME>-*[^\.d]; do mkdir -p "target/cov/$(basename $file)"; kcov --exclude-pattern=/.cargo,/usr/lib --verify "target/cov/$(basename $file)" "$file"; done &&
+  for file in target/debug/examplerust-*[^\.d]; do mkdir -p "target/cov/$(basename $file)"; ./kcov-build/usr/local/bin/kcov --exclude-pattern=/.cargo,/usr/lib --verify "target/cov/$(basename $file)" "$file"; done &&
   bash <(curl -s https://codecov.io/bash) &&
   echo "Uploaded code coverage"
 ```

